core/src/main/java/org/opensearch/sql/calcite/utils/PPLOperandTypes.java

```diff
@@ -128,6 +128,32 @@ private PPLOperandTypes() {}
               SqlTypeFamily.NUMERIC,
               SqlTypeFamily.NUMERIC,
               SqlTypeFamily.NUMERIC));
+  public static final UDFOperandMetadata WIDTH_BUCKET_OPERAND =
```

Comment:
np: in future we may need more readable function signature/checker spec or docs for internal function usage too.